### PR TITLE
CNV-5020 FOR 2.4: virtctl expose documentation should specify need to be in project

### PR DESCRIPTION
**Pull request by bgaydosrh**
Adding requested prereq to module.

See Jira story link below for content details.

Tagging @aburdenthehand to review peer review changes.

Test Build: http://file.bos.redhat.com/bgaydos/061820_5/virt/virtual_machines/virt-accessing-vm-consoles.html

Label: Peer Review Needed and enterprise-4.5. Another PR (https://github.com/openshift/openshift-docs/pull/22534) handles this change for enterprise-4.4.

Bob

### CODE REVIEW

**Comment by jameslabocki** [comment]
LGTM

**Comment by bgaydosrh** [comment]
> LGTM

Excellent Thanks @jameslabocki. Moving to QE review.

Bob

File: modules/virt-accessing-vmi-ssh.adoc [hunk 1]
```diff
@@ -14,6 +14,8 @@ the `fedora-vm-ssh` service that forwards port 22 of the `<fedora-vm>` virtual
 machine to a port on the node:
 
 .Prerequisites
+* You must be working in the {product-title} project in which the virtual machine
```
Comment (aburdenthehand) [inline modules/virt-accessing-vmi-ssh.adoc:17]: I don't think the product-title is necessary here, and it should be VMI for consistency. Suggest to shorten it to something like:
"You must be in the same project as the virtual machine instance."

Comment (bgaydosrh) [reply modules/virt-accessing-vmi-ssh.adoc]: Changed as suggested.

**Pull request merged**